docs/multi-stage-query/reference.md

```diff
@@ -366,27 +366,31 @@ Common properties to configure the behavior of durable storage
 |`druid.msq.intermediate.storage.maxRetry` | 10 | Optional. Defines the max number times to attempt S3 API calls to avoid failures due to transient errors. | 
 |`druid.msq.intermediate.storage.chunkSize` | 100MiB | Optional. Defines the size of each chunk to temporarily store in `druid.msq.intermediate.storage.tempDir`. The chunk size must be between 5 MiB and 5 GiB. A large chunk size reduces the API calls made to the durable storage, however it requires more disk space to store the temporary chunks. Druid uses a default of 100MiB if the value is not provided.| 
 
-Following properties need to be set in addition to the common properties to enable durable storage on S3
+To use S3 for durable storage, you also need to configure the following properties:
```

Comment:
Perhaps in the following tables, we can add a new column Required, and have yes/no in the values instead of adding `Required/Optional` in the description.